### PR TITLE
76 update edit flow

### PR DESCRIPTION
**Pull request by schultztimothy**
fixes: #76 

- [x] Add "cancel" button 
- [x] Remove IPFS confirmation message (should we get a lot of feedback that folks want to see their file on IPFS, we could bring this back with better UX) 
- [x] After publishing update, should redirect users to Project Details page (for the project they just created)

### CODE REVIEW

File: client/src/components/base/ProjectForm.tsx [hunk 1]
```diff
@@ -53,7 +59,8 @@ function ProjectForm({ currentGrantId }: { currentGrantId?: string }) {
     }
 
     await dispatch(saveFileToIPFS(formInputs, FileTypes.PROJECT));
-    dispatch(publishGrant(currentGrantId));
+    await dispatch(publishGrant(currentGrantId));
+    setTimeout(() => navigate(slugs.grants, { replace: true }), 1500);
```
Comment (schultztimothy) [inline client/src/components/base/ProjectForm.tsx:63]: Added a timeout so the project created notification shows for 1.5 seconds before they are redirected

Comment (gravityblast) [reply client/src/components/base/ProjectForm.tsx:63]: @schultztimothy  cool! do we need the `replace` there?

**Comment by gravityblast** [comment]
now we have a `cancel` button on the bottom, and an `exit` one on top, do we need both? /cc @schultztimothy @michellema1208

**Review by gravityblast — APPROVED** [review]
LGTM! just added a couple of comments.

File: client/src/components/base/ProjectForm.tsx (hunk 1 above)
Comment (schultztimothy) [reply client/src/components/base/ProjectForm.tsx]: Good call! I removed both instances of replace

**Pull request merged**